src/fastcs/backends/epics/ioc.py

```diff
@@ -224,33 +266,62 @@ async def async_write_display(value: T):
 
 
 def _get_output_record(pv: str, attribute: AttrW, on_update: Callable) -> Any:
+    attribute_fields = {}
+    if attribute.description is not None:
+        attribute_fields.update({"DESC": attribute.description})
     if attr_is_enum(attribute):
         assert attribute.allowed_values is not None and all(
             isinstance(v, str) for v in attribute.allowed_values
         )
         state_keys = dict(zip(MBB_STATE_FIELDS, attribute.allowed_values, strict=False))
-        return builder.mbbOut(pv, always_update=True, on_update=on_update, **state_keys)
+        return builder.mbbOut(
+            pv,
+            always_update=True,
+            on_update=on_update,
+            **state_keys,
+            **attribute_fields,
+        )
 
     match attribute.datatype:
-        case Bool(znam, onam):
-            return builder.boolOut(
+        case Bool():
+            record = builder.boolOut(
                 pv,
-                ZNAM=znam,
-                ONAM=onam,
+                **datatype_to_epics_fields(attribute.datatype),
                 always_update=True,
                 on_update=on_update,
             )
         case Int():
-            return builder.longOut(pv, always_update=True, on_update=on_update)
-        case Float(prec):
-            return builder.aOut(pv, always_update=True, on_update=on_update, PREC=prec)
+            record = builder.longOut(
+                pv,
+                always_update=True,
+                on_update=on_update,
+                **datatype_to_epics_fields(attribute.datatype),
+                **attribute_fields,
+            )
+        case Float():
+            record = builder.aOut(
+                pv,
+                always_update=True,
+                on_update=on_update,
+                **datatype_to_epics_fields(attribute.datatype),
+                **attribute_fields,
+            )
         case String():
-            return builder.longStringOut(pv, always_update=True, on_update=on_update)
+            record = builder.longStringOut(
+                pv, always_update=True, on_update=on_update, **attribute_fields
+            )
         case _:
             raise FastCSException(
                 f"Unsupported type {type(attribute.datatype)}: {attribute.datatype}"
             )
 
+    def datatype_updater(datatype: DataType):
+        for name, value in datatype_to_epics_fields(datatype).items():
+            record.set_field(name, value)
```

Comment:
I think this is important to get test coverage on.

Reply:
I added a test in the squash.

Reply:
But I'll extend it for all relevant `AttR` types.